server/embed/serve.go

```diff
@@ -301,7 +302,22 @@ func (sctx *serveCtx) registerGateway(dial func(ctx context.Context) (*grpc.Clie
 	if err != nil {
 		return nil, err
 	}
-	gwmux := gw.NewServeMux()
+
+	gwmux := gw.NewServeMux(
+		gw.WithMarshalerOption(gw.MIMEWildcard,
+			&gw.HTTPBodyMarshaler{
+				Marshaler: &gw.JSONPb{
```

Comment:
How did you arrived at those flags? Was it default in v1? Did you test them in any way?  Would be good to leave some comment.

Reply:
I will add comment, thx.

FYI. https://grpc-ecosystem.github.io/grpc-gateway/docs/mapping/customizing_your_gateway/

We have lots of test cases to verify the returned JSON structures.

Reply:
Sure, by tested I meant if you have tested what config is the best.

Reply:
Done.